atomic_reactor/plugins/post_group_manifests.py

```diff
@@ -164,7 +165,8 @@ def get_worker_manifest(self, worker_data):
 
                 url = '{0}/v2/{1}/manifests/{2}'.format(registry, repo, digest)
                 self.log.debug("attempting get from %s", url)
-                response = requests.get(url, **kwargs)
+                response = session.get(url, **kwargs)
+                response.raise_for_status()
```

Comment:
@mlangsdorf , we noticed that for this particular `get` API call the status code is not checked.
We're adding it now. Can you confirm this was not a deliberate decision?